### PR TITLE
(v2.0) GOLD-883: iOS 8.0 scroll hotfix

### PR DESCRIPTION
**Pull request by jvaill**
**Status:** Ready for review
**JIRA:** https://mobify.atlassian.net/browse/GOLD-883

### CODE REVIEW

File: build/mobify.js [hunk 1]
```diff
@@ -1721,6 +1721,47 @@ Capture.setElementContentFromString = function(el, htmlString) {
...
+    openingBodyTag = openingBodyTagAsDiv.replace(/^<div/, '<body');
+
+    var script =
+        "<script>" +
+        "  window.requestAnimationFrame(function() {" +
+        "    window.requestAnimationFrame(function() {" +
+        "      document.body.style.display = null;" +
+        "    });" +
+        "  });" +
+        "</script>";
```
Comment (jansepar) [inline build/mobify.js]: `<\/script>` :). Just incase.

File: build/mobify.js [hunk 2]
```diff
@@ -1721,6 +1721,47 @@ Capture.setElementContentFromString = function(el, htmlString) {
...
+
+    htmlString = htmlString.replace(BODY_REGEX, openingBodyTag + script);
+    return htmlString;
```
Comment (jansepar) [inline build/mobify.js]: We could just change this to `return htmlString.replace(BODY_REGEX, openingBodyTag + script);`. Gotta save those LOC :)

**Comment by jansepar** [comment]
This looks good. Needs tests!

File: build/mobify.js [hunk 3]
```diff
@@ -1722,6 +1722,68 @@ Capture.setElementContentFromString = function(el, htmlString) {
 };
 
 /**
+ * iOS 8.0 has a bug where dynamically switching the viewport (by swapping the
+ * viewport meta tag) causes the viewport to automatically scroll. When
+ * capturing, the initial document never has an active meta viewport tag.
+ * Then, the rendered document injects one causing the aforementioned scroll.
+ *
+ * This patches HTML to hide the body until the first paint (and hopefully after
```
Comment (jansepar) [inline build/mobify.js:1736]: Maybe mention in this comment that this fix is only applied for iOS8 devices.

**Comment by jvaill** [comment]
Verbal :+1: from @jansepar

**Pull request merged**